.github/workflows/arm-main-cmake.yml

```diff
@@ -81,7 +80,6 @@ jobs:
             parallel: OFF
             mirror_vfd: ON
             direct_vfd: ON
-            ros3_vfd: ON
```

Comment:
The ROS3 VFD has been turned off now for all existing places in workflows where it was enabled, as it's no longer built with libcurl. Rather, a special workflow will be created in a followup PR to build aws-c-s3 and test with s3proxy.